### DISCUSSION TITLE
MockingBird ALL IN ONE！

### DISCUSSION THREAD

**Discussion by kslz** [babysor/MockingBird#793] [Show and tell]
我用便携版python装了所有的环境，外加打包了git、ffmpeg、样例数据集、N个声学模型 再写了所有常用功能的bat，现在任何小白都能解压即玩了
![image](https://user-images.githubusercontent.com/54951765/204806826-aec2de40-1a61-472a-bc22-c1ac5ef08ec6.png)
下载地址：https://pan.baidu.com/s/1-aIztnVVmp2QzYj4rgZ0og?pwd=6nu9 提取码：6nu9

**Comment by Leiou1939** [comment]
感谢

**Reply by dasi852852** [reply]
יש כאן דוברי עברית?
